SMelidoni/personal-finance-manager
# feature/add-uuid-to-user

## Description of Changes

Implemented UUID as primary key for User entity. This provides a more secure and scalable approach to user identification in the database.

## Type of Change

- Bug Fix
- Breaking Change
- Documentation Update
- Other

## Describe the problem this PR solves and your solution.

Previously, the user ID was a sequentially generated integer. This change to UUID provides enhanced security as it is harder to guess and more scalable due to its universally unique nature.

## Testing Steps

1. Create a new user in the application.
2. Check in the database that the new user has a UUID in the `id` field.

## Screenshots

N/A

## Checklist
